apps/aecore/src/aec_sync.erl

```diff
@@ -203,16 +203,20 @@ handle_cast(_, State) ->
 handle_info({gproc_ps_event, Event, #{info := Info}},
             State = #state{ gossip_txs = GossipTxs }) ->
     %% FUTURE: Forward blocks only to outbound connections.
-    PeerIds = [ aec_peers:peer_id(P) || P <- aec_peers:connected_peers() ],
+    %% Take a random subset (possibly empty) of peers that agree with us 
+    %% on chain height to forward blocks and transactions to.
+    PeerIds = [ aec_peers:peer_id(P) || P <- aec_peers:get_random(10) ],
     NonSyncingPeerIds = [ P || P <- PeerIds, not peer_in_sync(State, P) ],
```

Comment:
we did gossip before to peers that we are in sync with - so commit message is not accurate?
He we limit it to 10 random nodes - maybe we should make it configurable?

Reply:
Ah, yes, sorry, blindness doesn't hurt. And it makes sense of course to sync with nodes that have different view of the chain.

Not really convinced with no-config way. We don't protect it with protocol, so there will be number of implementations trying out another values. We can expose it and make it easier for testing.

Reply:
Otherwise the variable would have been named `NotSyncedPeerIds` obviously 🤷‍♂️